test/decode.js

```diff
@@ -11,11 +11,11 @@ test('decode: valid magnet uris', t => {
   t.equal(result.dn, 'Leaves of Grass by Walt Whitman.epub')
   t.equal(result.infoHash, 'd2474e86c95b19b8bcfdb92bc12c9d44667cfa36')
   const announce = [
-    'udp://tracker.example1.com:1337',
-    'udp://tracker.example2.com:80',
-    'udp://tracker.example3.com:6969',
     'udp://tracker.example4.com:80',
-    'udp://tracker.example5.com:80'
+    'udp://tracker.example5.com:80',
+    'udp://tracker.example3.com:6969',
+    'udp://tracker.example2.com:80',
+    'udp://tracker.example1.com:1337'
   ]
   t.deepEqual(result.tr, announce)
   t.deepEqual(result.announce, announce)
```

Comment:
Instead of reordering the array, how about doing `t.deepEqual(result.announce.sort(), announce.sort())`?

Although sorting the lists adds complexity, since the order is not guaranteed by the spec I think it's more clear if the array `announce` has an arbitrary order, instead of the order from the magnet link.

Reply:
yeah, better to just ignore the order in the tests if it doesn't matter